### PR TITLE
Add eslint

### PR DESCRIPTION
**Pull request by darenang44**

### CODE REVIEW

**Comment by underaj** [comment]
LGTM

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "rules": {
```
Comment (underaj) [inline .eslintrc]: You probably need this:
 "extends": "angular"

for your linter angular linter to work

Comment (underaj) [inline .eslintrc]: You probably need this:
"extends": "angular"

for your linter angular linter to work

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (underaj) [inline .eslintrc:1]: You probably need this:
"extends": "angular"

for your linter angular linter to work

Comment (darenang44) [inline .eslintrc:1]: Done

**Pull request merged**